ploparraymax.yaml

```diff
@@ -0,0 +1,3 @@
+- prop1: ded
+- prop2: ded
+- prop3: plop
```

Comment:
Is thie file needed anywhere or was it just used to test?

Reply:
just for test

Reply:
I shouldn't have pushed it!!! my bad. Thanks for pointing it